crates/ruff_linter/src/checkers/imports.rs

```diff
@@ -18,6 +20,7 @@ use crate::Locator;
 
 #[allow(clippy::too_many_arguments)]
 pub(crate) fn check_imports(
+    path: &Path,
```

Comment:
I'd prefer if we pass the `PythonVersion` here rather than the `Path`. It currently feels inconsistent where almost all rules take a `PythonVersion` but there are a few exceptions where we pass the `Path` and the rule has to figure out how to get the target version.